vNext/extensions/applicationinsights-analytics-js/src/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -345,6 +353,28 @@ export class ApplicationInsights implements IAppInsights, ITelemetryPlugin, IApp
         }
     }
 
+    /**
+    * @ignore INTERNAL ONLY
+    * @param exception
+    * @param properties
+    * @param systemProperties
+    */
+    public sendExceptionInternal(exception: IExceptionTelemetry, customProperties?: { [key: string]: any }, systemProperties?: { [key: string]: any }) {
```

Comment:
same here, does 1P require systemproperties to be passed in for exception telemetry?

Reply:
Yes, is one of the reasons of this PR